health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/BannerCard.js

```
@@ -0,0 +1,136 @@
+  const renderMetrics = () => {
+    if (chartData?.length === 0) {
+      return (
+        <div style={{ margin: "0px auto" }}>
+          <NoData t={t} />
+        </div>
+      );
+    }
+
+    return chartData.map((data) => {
+      return (
+        <div className="digit-banner-table tooltip">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)